src/open_inwoner/ssd/tests/test_views.py

```diff
@@ -87,7 +87,7 @@ def test_uitkering_post_fail(self, mock_report):
         self.assertEqual(response.status_code, HTTPStatus.OK)
 
         self.assertContains(
-            response, "Geen uitkeringsspecificatie gevonden voor Dec 1985"
+            response, "Geen uitkeringsspecificatie gevonden voor december 1985"
```

Comment:
In the taiga issue I see that the title of the `Maandspecificatie` pdf is what should be fixed, so I would expect that to be addressed too?

Reply:
This was resolved last year, see [https://github.com/maykinmedia/open-inwoner/blob/fix/1808-maandspecificatie-header/src/open_inwoner/ssd/client.py#L192](https://github.com/maykinmedia/open-inwoner/blob/fix/1808-maandspecificatie-header/src/open_inwoner/ssd/client.py#L192)